cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/servlet/UserTaskManager.java

```diff
@@ -159,13 +163,14 @@ protected boolean removeEldestEntry(Map.Entry<UUID, UserTaskInfo> eldest) {
    *                 returns a completed Future.
    * @param step The index of the step that has to be added or fetched.
    * @param isAsyncRequest Indicate whether the task is async or sync.
+   * @param parameters Help {@link UserTaskInfo} determine the response to persist.
```

Comment:
A JavaDoc should indicate what the `parameters` is -- i.e. not how it is intended to be used from outside the class. Also, do we really need to use parameters in this function, can't we use the parameter map?